src/components/App.js

```diff
@@ -3,60 +3,60 @@ import Navbar from "./Navbar";
 import Homapage from "./Homapage";
 import Profile from "./Profile";
 import CreateNewPin from "./CreateNewPin";
+import Login from "./Login";
+import React, {useState, useEffect} from "react";
 import {BrowserRouter, Routes, Route} from "react-router-dom";
-
-import {initializeApp} from "firebase/app";
-import {
-  getFirestore,
-  collection,
-  doc,
-  getDocs,
-  getDoc,
-} from "firebase/firestore";
-
-const firebaseConfig = {
-  apiKey: process.env.REACT_APP_FIREBASE_APIKEY,
-  authDomain: process.env.REACT_APP_FIREBASE_DOMAIN,
-  projectId: process.env.REACT_APP_FIREBASE_PROJECTID,
-  storageBucket: process.env.REACT_APP_FIREBASE_STORAGEBUCKET,
-  messagingSenderId: process.env.REACT_APP_FIREBASE_MESSAGINGSENDERID,
-  appId: process.env.REACT_APP_FIREBASE_APPID,
-  measurementId: process.env.REACT_APP_FIREBASE_MEASUREMENTID,
-};
+import {getAuth, onAuthStateChanged} from "firebase/auth";
 
 function App() {
-  const app = initializeApp(firebaseConfig);
-  const db = getFirestore();
-
-  // line 41-59 is testing getting from firebase
-  // will be used later when developing functions
-  const getUser = async (id) => {
-    const user = await getDoc(doc(db, "user", id));
-    if (!user.exists()) {
-      console.error("Note doesn't exist");
-      return;
-    }
+  const [login, setLogin] = useState(false);
+  const [uid, setUid] = useState();
 
-    return user.data();
-  };
-  // getUser("M49BbsijmzC2W5TxBbg2");
+  useEffect(() => {
+    const auth = getAuth();
 
-  const getPins = async () => {
-    const notesSnapshot = await getDocs(collection(db, "pin"));
-    const pins = notesSnapshot.docs.map((doc) => doc.data());
+    onAuthStateChanged(auth, (user) => {
+      if (!user) {
+        console.log("not login");
 
-    return pins;
-  };
-  // getPins();
+        return;
+      }
+      const uid = user.uid;
+      console.log("login", uid);
```

Comment:
Clear log...

Reply:
I keep this console for knowing login status; it'll be removed before release.